.github/workflows/bench.yml

```diff
@@ -0,0 +1,21 @@
+name: Bench
+on:
+    pull_request:
+    push:
+        branches:
+            - main
+
+permissions:
+    contents: write
+    deployments: write
+
+jobs:
+    benchmark:
+        name: Run Rust benchmark 
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - run: rustup toolchain update nightly && rustup default nightly
+            - name: Run benchmark
+              run: cd benches && rm -r output.txt && touch output.txt && cargo +nightly bench | tee output.txt
```

Comment:
what would happen on merge to main for the output.txt?

Reply:
Nothing. It would remain like that, but for every push to mina amd PR merged `output.txt` would be removed and the new benchmarks written to it

Reply:
👍